resources/views/filament/alerts/alert-banner.blade.php

```diff
@@ -0,0 +1,43 @@
+@props(['alertBanner'])
+
+@php
+    if ($alertBanner) {
+        $status = $alertBanner['status'];
+
+        $title = $alertBanner['title'];
+        $body = $alertBanner['body'];
+
+        $icon = $alertBanner['icon'] ?? match ($status) {
+            "success" => "tabler-circle-check",
+            "warning" => "tabler-exclamation-circle",
+            "danger" => "tabler-circle-x",
+            default => "tabler-info-circle",
+        };
+
+        $colorClasses = match ($status) {
+            "success" => "text-success-600 dark:text-success-500",
+            "warning" => "text-warning-600 dark:text-warning-500",
+            "danger" => "text-danger-600 dark:text-danger-500",
+            default => "text-info-600 dark:text-info-500",
+        };
```

Comment:
Alternatively, can we just move this under enum methods so we don't have a `@php` block in our blade files.